main/src/main/java/org/mobilitydata/gtfsvalidator/validator/ShapeIncreasingDistanceValidator.java

```diff
@@ -225,4 +236,59 @@ static class EqualShapeDistanceDiffCoordinatesNotice extends ValidationNotice {
           getDistanceMeters(current.shapePtLatLon(), previous.shapePtLatLon());
     }
   }
+
+  /**
+   * Two consecutive points have equal `shape_dist_traveled` and different lat/lon coordinates in
+   * `shapes.txt` and the distance between the two points is less than 1.11m.
```

Comment:
```suggestion
   * `shapes.txt` and the distance between the two points is less more than 0 and than 1.11m.
```

Reply:
@jcpitre do you mean "is more than 0 and less than 1.11m"?

Reply:
isn't a distance always greater than 0 anyway?

Reply:
No, but I guess people will understand.